<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settng_Lubrication</class>
 <widget class="QWidget" name="Settng_Lubrication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color:black;
}
</string>
  </property>
  <widget class="QLabel" name="LBL_Tips1">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:16px;</string>
   </property>
   <property name="text">
    <string>间隔距离</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Tips2">
   <property name="geometry">
    <rect>
     <x>466</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:16px;</string>
   </property>
   <property name="text">
    <string>喷脂时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_DisReturnValue">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>80</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:16px;</string>
   </property>
   <property name="text">
    <string>间隔距离返回值: 0m</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_TimeReturnValue">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>80</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:16px;</string>
   </property>
   <property name="text">
    <string>喷脂时间返回值: 0s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_DisValue">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>180</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid white;
color:white;</string>
   </property>
   <property name="text">
    <string>250</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_TimeValue">
   <property name="geometry">
    <rect>
     <x>438</x>
     <y>180</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid white;
color:white;</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DisAdd">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>120</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/LubAdd.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DisSub">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>230</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/LubSub.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_TimeAdd">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>120</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/LubAdd.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_TimeSub">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>230</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/LubSub.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_DisMAXValue">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>340</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:14px;</string>
   </property>
   <property name="text">
    <string>最大值          500m</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_DisMINValue">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>380</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:14px;</string>
   </property>
   <property name="text">
    <string>最小值          50m</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_DisEachValue">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>420</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:14px;</string>
   </property>
   <property name="text">
    <string>每次调整      50m</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_TimeEachValue">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:14px;</string>
   </property>
   <property name="text">
    <string>每次调整     1s</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_TimeMAXValue">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:14px;</string>
   </property>
   <property name="text">
    <string>最大值         15.0s</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_TimeMINValue">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>380</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:14px;</string>
   </property>
   <property name="text">
    <string>最小值         10.0s</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
